llvm/lib/Target/BPF/BPFFrameLowering.cpp

```diff
@@ -20,13 +20,44 @@
 
 using namespace llvm;
 
+namespace {
+
+void adjustStackPointer(MachineFunction &MF, MachineBasicBlock &MBB,
+                        MachineBasicBlock::iterator &MBBI,
+                        unsigned int Opcode) {
+  MachineFrameInfo &MFI = MF.getFrameInfo();
+  int NumBytes = (int)MFI.getStackSize();
```

Comment:
Is the total stack size known to llvm or does it allow overflow expecting the runtime to catch it?

Reply:
It can't be known statically and the runtime needs to handle overflows anyway for hand rolled code